README.md

```diff
@@ -134,29 +134,75 @@ Sketches:
 --------------------------------------------------------------------------------
 <BASE64_ENCODED_DATA_POINTS|REFERENCE_TO_OTHER_DATA_FILE>
 ```
-
-Each Protocol has unique set of parameters. For example:
-- CPI
+Each Sync Protocol has an integer to identify it. The list of integers is as follows:
```

Comment:
The risk in listing specific integers for each sync is that the documentation will have to be updated each time the sync list is modified.  Is there, perhaps, a file to which you can point that lists the current numbers associated with
each sync?

Reply:
In GenSync.h, there is a list of sync protocols that correspond to each number. However, it is a bit unclear in the documentation of GenSync.h that those are the protocols. Instead of including it here, maybe I should update the documentation in GenSync.h?